fapi-client/src/main/scala/com/gu/facia/api/contentapi/ContentApi.scala

```diff
@@ -34,7 +34,7 @@ object ContentApi extends StrictLogging {
   }
 
   def getHydrateResponse(client: ContentApiClient, searchQueries: Seq[SearchQuery])(implicit ec: ExecutionContext): Response[Seq[SearchResponse]] = {
-    Response.Async.Right(Future.traverse(searchQueries)(client.getResponse)) mapError { err =>
+    Response.Async.Right(Future.traverse(searchQueries)(client.getResponse(_))) mapError { err =>
```

Comment:
I think the addition of an implicit parameter group turned that function into a method.